tests/unittests/net/network_configs.py

```diff
@@ -2586,22 +3729,22 @@
                 method=manual
                 may-fail=false
                 address1=192.168.0.2/24
-                gateway=192.168.0.1
-                route1=10.1.3.0/24,192.168.0.3
+                route1=0.0.0.0/0,192.168.0.1
```

Comment:
is this change intentional? (repeat for the next several changes)

Reply:
Yes, like the miimon v2 renders slightly differently and github is doing a poor job rendering it. v2 renders a 0.0.0.0 route rather than gateway, but functionally it is the same.

![image](https://github.com/canonical/cloud-init/assets/153674/8454d7c0-71cf-4860-8182-f9ec98cd52d9)